app/routes/__index/certificate/index.tsx

```diff
@@ -85,6 +87,15 @@ export default function CertificateIndexRoute() {
     certificate?.status === 'pending' ? 15_000 : null
   );
 
+  const canRenew = useMemo((): boolean => {
```

Comment:
I would rename this to `isRenewable` too, so it matches everything else you're doing.